modelbaker/dbconnector/db_connector.py

```diff
@@ -343,6 +343,12 @@ def get_classes_relevance(self):
         """
         return []
 
+    def multiple_geometry_tables(self):
+        """
+        Returns a list of tables having multiple geometry columns.
+        """
```

Comment:
Perhaps add a note stating that this is only useful or expected for GPKG?
That could explain why we can return an empty list for PG and MSSQL.